interpreter/python/python.go

```diff
@@ -753,6 +753,8 @@ func Loader(ebpf interpreter.EbpfHandler, info *interpreter.LoaderInfo) (interpr
 	// The Python main interpreter loop history in CPython git is:
 	//
 	//nolint:lll
+	// 47ad32d158e v3.13 2024-10-10 _PyEval_EvalFrameDefault(PyThreadState*,_PyInterpreterFrame*,int)
+	// 01daccff3c2 v3.12 2024-10-10 _PyEval_EvalFrameDefault(PyThreadState*,_PyInterpreterFrame*,int)
```

Comment:
The commits refer here to unrelated changes. And I think the intent here is to list only the commits that actually change the function prototype or name somehow. Was there some relevant changes?

Reply:
It is not clear to me, how the commits are selected. E.g. from 3.9 to 3.10 there is no change in the function signature, but two different commits are referenced. 
As there was no change in the function signature, it might be ok, to just drop the lines for 3.12 and 3.13?

Reply:
``` diff
-       // deaf509e8fc v3.11 2022-11-15 _PyEval_EvalFrameDefault(PyThreadState*,_PyInterpreterFrame*,int)
-       // bc2cdfc8157 v3.10 2022-11-15 _PyEval_EvalFrameDefault(PyThreadState*,PyFrameObject*,int)
+       // 87af12bff33 v3.11 2022-02-15 _PyEval_EvalFrameDefault(PyThreadState*,_PyInterpreterFrame*,int)
+       // ae0a2b75625 v3.11 2021-06-25 _PyEval_EvalFrameDefault(PyThreadState*, _interpreter_frame*,int)
```

In case you want to fix the history at this time.

Reply:
applied the suggested changes.